config/sync/user.role.site_embedder.yml

```diff
@@ -0,0 +1,159 @@
+uuid: 9dedbe8e-27b6-40de-95fd-29e4efe67be4
+langcode: en
+status: true
+dependencies: {  }
+id: site_embedder
+label: 'Site Embedder'
+weight: 1
+is_admin: null
+permissions:
```

Comment:
This role should be additive to the other roles. Its only permissions. Can be the oembed field. Then a user would be given the site manager and the embeddable roles together

Reply:
fixed.